src/controllers/forgotpassword.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken'
+import { Employee } from '../../models'
+import nodemailer from 'nodemailer'
+
+// Setting a transporter
+const transporter = nodemailer.createTransport({
+  // host and port for MailHog
+  host: process.env.TRANSPORTER_HOST,
+  port: process.env.TRANSPORTER_PORT
+})
+
+// forgotpassword controller function
+const forgotpassword = (req, res) => {
+  const { email } = req.body
+  // Responses for translations
+  const responseNoUserAvailable = req.t('forgot_password.failure')
+  const responseUserAvailable = req.t('forgot_password.success')
+  Employee.findOne({
+    where: {
+      email: email
+    }
+  }).then((user) => {
+    if (!user) {
+      res.status(400).json({
+        status: 400,
+        message: `${responseNoUserAvailable}`
+      })
+    } else {
+      // Generating Token for resetting password
+      const resetToken = jwt.sign({ email: email }, process.env.RESET_SECRET, { expiresIn: process.env.RESET_TOKEN_EXPIRES })
+      res.cookie('jwt', resetToken, { httpOnly: true, expiresIn: process.env.RESET_TOKEN_EXPIRES })
+
+      // Reset password url
+      const url = `http://${process.env.HOST_NAME}:${process.env.APP_PORT}/reset/${resetToken}`
```

Comment:
Please make this as a whole variable, do you think we will be having `HTTP` in production or `HTTPS` that is why you don't need to break the base URL... when we say base URL we mean the unchanging part which you add on the route